packages/discovery-react-components/src/components/DocumentPreview/__stories__/DocumentPreview.stories.tsx

```diff
@@ -64,3 +98,26 @@ function passageSelection(doc: QueryResult, passages: object): QueryResult {
     document_passages: [passages[passageType]]
   };
 }
+
+function tableSelection(doc: QueryResult): QueryResult {
```

Comment:
I added this back in as well, since it was there the last time we had the PDF view. However, it is not currently working. Needs to be investigated.